receiver/awsxrayreceiver/README.md

```diff
@@ -0,0 +1,68 @@
+# AWS X-Ray Receiver
+
+**Status: alpha**
+
+## Overview
+The AWS X-Ray receiver accepts segments (i.e. spans) in the [X-Ray Segment format](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html).
+This enables the collector to receive spans emitted by the existing X-Ray SDK. [Centralized sampling](https://github.com/aws/aws-xray-daemon/blob/master/CHANGELOG.md#300-2018-08-28) is also supported via a local TCP port.
+
+The requests sent to AWS are authenticated using the mechanism documented [here](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+
+## Configuration
+
+Example:
+
+```yaml
+receivers:
+  aws_xray:
+    endpoint: localhost:2000
+    proxy_server:
+      tcp_endpoint: localhost:2000
+      proxy_address: ""
+      insecure: false
+      server_name_override: ""
+      region: ""
+      role_arn: ""
+      aws_endpoint: ""
+      local_mode: false
+```
+
+The default configurations below are based on the [default configurations](https://github.com/aws/aws-xray-daemon/blob/master/pkg/cfg/cfg.go#L99) of the existing X-Ray Daemon.
+
+### endpoint (Optional)
+The UDP address and port on which this receiver listens for X-Ray segment documents emitted by the X-Ray SDK.
+
+Default: `localhost:2000`
+
+### proxy_server (Optional)
+Defines configurations related to the local TCP proxy server.
+
+### tcp_endpoint (Optional)
+The address and port on which this receiver listens for calls from the X-Ray SDK and relays them to the AWS X-Ray backend to get sampling rules and report sampling statistics.
+
+Default: `localhost:2000`
```

Comment:
The defaults for all other receivers were recently changed to 0.0.0.0. To make it uniform it is best to use it here as well.